jira/client.py

```diff
@@ -4062,35 +4062,35 @@ def update_sprint(self, id, name=None, startDate=None, endDate=None, state=None)
 
         return json_loads(r)
 
-    def incompletedIssuesEstimateSum(self, board_id, sprint_id):
-        """Return the total incompleted points this sprint."""
+    def sprint_report(self, board_id, sprint_id):
+        """Return the sprint report."""
         return self._get_json(
             "rapid/charts/sprintreport?rapidViewId=%s&sprintId=%s"
             % (board_id, sprint_id),
             base=self.AGILE_BASE_URL,
-        )["contents"]["incompletedIssuesEstimateSum"]["value"]
+        )["contents"]
+
+    def incompletedIssuesEstimateSum(self, board_id, sprint_id):
```

Comment:
Just a quick question.  how come the code is mixing camel case and snake case for method names?  AFAIK python convention is to use snake case for the method names.

Reply:
Not sure.  I'm just trying to stick to the current convention in `client.py`